### PR TITLE
pkam tool changes to accept .keys or .zip file as input

### PR DESCRIPTION
**Pull request by naresh0689**

### CODE REVIEW

File: at_pkam/bin/main.dart [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  var filePath = arguments[0];
+  var challenge = arguments[1];
+  var privateKey;
+  if (filePath.endsWith(AT_KEYS)) {
+    privateKey = await getSecretFromAtKeys(filePath);
+  } else if (filePath.endsWith(ZIP)) {
+    privateKey = await getSecretFromZip(filePath);
+  } else {
+    throw Exception(
+        'Provide .keys or zip file. \n Syntax : dart main.dart <.keys/zip file path> <from_response>');
```
Comment (murali-shris) [inline at_pkam/bin/main.dart:18]: you have to print usage instead of passing message in exception

Comment (murali-shris) [reply at_pkam/bin/main.dart:18]: use argparser util that is used in root/secondary server startup

File: at_pkam/lib/commandline_parser.dart [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    }
+  }
+
+  ArgParser getParser() {
+    var parser = ArgParser();
+    parser.addOption('file_path',
+        abbr: 'p', help: '.keys or .zip file path which contains keys');
+    parser.addOption('from_response', abbr: 'r', help: 'from:@atSign response');
+    parser.addOption('aes_key',
+        abbr: 'a', help: 'aes key file path', defaultsTo: '');
```
Comment (murali-shris) [inline at_pkam/lib/commandline_parser.dart:31]: aes key file path or just aes key ?

Comment (naresh0689) [reply at_pkam/lib/commandline_parser.dart:31]: file path

**Review by murali-shris — APPROVED** [review]

**Pull request merged**